docs/framework/wcf/feature-details/how-to-secure-messages-within-reliable-sessions.md

```diff
@@ -15,52 +15,54 @@ author: "BrucePerlerMS"
+
+For the source copy of the example on which this configuration procedure is based, see the [WS Reliable Session](../../../../docs/framework/wcf/samples/ws-reliable-session.md).
+
+The essential items of the second task are accomplished by setting the `mode` attribute of the **\<security>** element contained in the **\<binding>** element of the client and service to `Message`.
+
+The essential items of the third task are accomplished by setting the `clientCredentialType` attribute of the **\<message>** element contained in the **\<security>** element of the client and service to `Certificate`.
+
 > [!NOTE]
->  When using message security with reliable sessions, if the client is not authenticated, Reliable Messaging attempts to authenticate the client until a timeout occurs instead of throwing an exception upon first failure.  
-  
-### To configure the service with a WSHttpBinding to use a reliable session  
-  
-1.  This procedure is described in [How to: Exchange Messages Within a Reliable Session](../../../../docs/framework/wcf/feature-details/how-to-exchange-messages-within-a-reliable-session.md).  
-  
-### To configure the client with a WSHttpBinding to use a reliable session  
-  
-1.  This procedure is described in [How to: Exchange Messages Within a Reliable Session](../../../../docs/framework/wcf/feature-details/how-to-exchange-messages-within-a-reliable-session.md).  
-  
-### To set the mode and ClientCredentialType in configuration  
-  
-1.  Add an appropriate binding element to the [\<bindings>](../../../../docs/framework/configure-apps/file-schema/wcf/bindings.md) element of the configuration file. The following example adds a [\<wsHttpBinding>](../../../../docs/framework/configure-apps/file-schema/wcf/wshttpbinding.md) element.  
-  
-2.  Add a <`binding`> element and set its `name` attribute to an appropriate value.  
-  
-3.  Add a `<security>` element and set the `mode` attribute to `Message`.  
-  
-     The following example sets the mode to `Message`, and then sets the `clientCredentialType` attribute of the <`message`> element to `Certificate`.  
-  
-    ```xml  
-    <wsHttpBinding>  
-    <binding name="MessageSecurity">  
-        <security mode="Message" />  
-           <message clientCredentialType = " Certificate" />  
-        </security>  
-    </binding>  
-    </wsHttpBinding >  
-    ```
\ No newline at end of file
+> When using message security with reliable sessions and if the client is not authenticated, Reliable Messaging attempts to authenticate the client until a timeout occurs instead of throwing an exception upon first failure.
```

Comment:
instead of "and", I'd use a comma, or rephrase this to "When using message security with reliable sessions, Reliable Messaging attempts to authenticate an unauthenticated client..."